dev/openchpl_views.sql

```diff
@@ -1314,6 +1316,30 @@ LEFT JOIN (SELECT string_agg(certification_body_id::text||':'||name, '|') as acb
 				) dev_acb_map_inner 
 		GROUP BY vendor_id) dev_acb_map4
 	    ON dev_acb_map4.vendor_id = dev.vendor_id
+LEFT JOIN (select vendor_id, string_agg(certification_criterion_id::text, '|') delimited_criteria_all_listings
+			from (select distinct cpd.vendor_id, cc.certification_criterion_id 
+					from openchpl.certification_criterion cc 
+						inner join openchpl.certification_result cr 
+							on cc.certification_criterion_id = cr.certification_criterion_id
+						inner join openchpl.certified_product_details cpd 
+							on cr.certified_product_id = cpd.certified_product_id
+					where cc.deleted = false
+					and cr.deleted = false) dev
```

Comment:
I usually include `cr.success = true` when I'm getting cert results. I'm not 100% sure it's always necessary, but it might be?